genmod/commands/analyze.py

```diff
@@ -11,226 +11,220 @@
+        "LOC100288162",
+        "SRGAP2D",
+        "FAM272C",
+        "SNAR-A3",
+        "SNAR-A4",
+        "SNAR-A5",
+        "SNAR-A6",
+        "SNAR-A7",
+        "SNAR-A8",
+        "SNAR-A9",
+        "SNAR-A10",
+        "SNAR-A11",
+        "SNAR-A14",
+        "GLUD1P7",
+    ]
+)
+
 
 def check_families(variant_file):
     """Loop through the vcf file and check which families that are found."""
     families = set([])
-    if variant_file == '-':
-        variant_parser = VCFParser(fsock = sys.stdin)
+    if variant_file == "-":
+        variant_parser = VCFParser(fsock=sys.stdin)
     else:
-        variant_parser = VCFParser(infile = variant_file)
+        variant_parser = VCFParser(infile=variant_file)
     for variant in variant_parser:
-        genetic_models = variant['info_dict'].get('GeneticModels', None)
+        genetic_models = variant["info_dict"].get("GeneticModels", None)
         if genetic_models:
             for family_models in genetic_models:
-                family = family_models.split(':')[0]
+                family = family_models.split(":")[0]
                 families.add(family)
     return families
 
 
-def print_results(variant_dict, 
-                    outfile, 
-                    vcf_header,
-                    family_id, 
-                    score_key='CADD', 
-                    freq_key='1000G_freq', 
-                    mode = 'homozygote', 
-                    silent=False):
+def print_results(
+    variant_dict,
+    outfile,
+    vcf_header,
+    family_id,
+    score_key="CADD",
+    freq_key="1000G_freq",
+    mode="homozygote",
+    silent=False,
+):
     """Print the variants to a results file or stdout."""
-    
-    score_dict = {} # A dictionary with {variant_id: score}. Score is usually cadd score or rank score
+
+    score_dict = {}  # A dictionary with {variant_id: score}. Score is usually cadd score or rank score
     # for variant_id, variant in sorted(variant_dict.items(), key = lambda sort_key: float(sort_key[1]['info_dict'].get('CADD', '0')), reverse=True):
     column_width = 12
     length_of_output = 20
     for variant_id in variant_dict:
         # Get the score for each variant:
         max_score = max(
-                            [
-                                float(score) for score in 
-                                variant_dict[variant_id]['info_dict'].get(
-                                                                    score_key, 
-                                                                    '0')
-                            ]
-                        )
-        if mode == 'compound':
+            [float(score) for score in variant_dict[variant_id]["info_dict"].get(score_key, "0")]
+        )
+        if mode == "compound":
             # If we look at compounds we want to consider the combined score
-            family_compounds = compound_dict[variant_id]['info_dict'].get('Compounds', None)
-            if compounds:
```

Comment:
Did this work as intended before changing to family_compounds?

Reply:
No, no way. I'm guessing `analyze` was ever only used by Måns or possibly some more students, ~pre `Puzzle`, and `Scout` still at v1 or so. It would work for e.g. non-compound ARs and for AD dn but not other de novo.